sources/packages/web/src/components/layouts/student/StudentPageContainer.vue

```diff
@@ -32,6 +33,11 @@ export default defineComponent({
       required: false,
       default: false,
     },
+    containerClass: {
```

Comment:
The idea of the `full-page-container` is to avoid external customizations as much as possible centralizing the "different common patterns" in the `layoutTemplate` property.
It means, IMO, if we need a new "container style" it would be good to have a new `layoutTemplate` type for it because it can be used in other places.
That said, is there a UI definition that states that this page is not following the patterns already in place?

Reply:
OK. I'm gonna try the existing layouts or create a new one then. According to Figma the container needs to have a max-width of 730px.

Reply:
We do not follow Figma-specific width in this way. Everything there will be some sort of fixed width.
We can check with @hellolynn-tbtb but my guess is that this page is just a centered container in the same way as the others 😉

Reply:
yes, but I talked to @hellolynn-tbtb about it and she said to make it to max-width = 730px...

Reply:
I talked with @hellolynn-tbtb, and we are not going to introduce a new pattern right now. Please use the centered layout already present.